workers/social/lib/social/models/session.coffee

```diff
@@ -57,6 +61,10 @@ module.exports = class JSession extends Model
           console.error err  if err?
 
 
+  @activeSession = secure (client, callback) ->
+    @one { clientId: client.sessionToken }, callback
```

Comment:
ah sorry, I thought you were asking for some place else, you're right I can add that check here. 👊